### PR TITLE
updated imports

### PR DESCRIPTION
**Pull request by sabrinatoro**
to add 1 new genes and some NCBITaxon required for annotations

### CODE REVIEW

File: src/ontology/imports/merged_import.owl [hunk 1]
```diff
@@ -25065,9 +25224,7 @@ AnnotationAssertion(<http://purl.obolibrary.org/obo/IAO_0000116> <http://purl.ob
 AnnotationAssertion(<http://purl.obolibrary.org/obo/IAO_0000116> <http://purl.obolibrary.org/obo/RO_0001025> "Most location relations will only hold at certain times, but this is difficult to specify in OWL. See http://purl.obolibrary.org/obo/ro/docs/temporal-semantics/"@en)
 AnnotationAssertion(rdfs:label <http://purl.obolibrary.org/obo/RO_0001025> "located in"@en)
 TransitiveObjectProperty(<http://purl.obolibrary.org/obo/RO_0001025>)
-ObjectPropertyDomain(Annotation(<http://purl.obolibrary.org/obo/IAO_0000116> "This is redundant with the more specific 'independent and not spatial region' constraint. We leave in the redundant axiom for use with reasoners that do not use negation.") <http://purl.obolibrary.org/obo/RO_0001025> <http://purl.obolibrary.org/obo/BFO_0000004>)
```
Comment (katiermullen) [inline src/ontology/imports/merged_import.owl:25068]: I don't know what this means, but I assume @sabrinatoro reviewed these ObjectPropertyDomain Annotations

**Comment by katiermullen** [comment]
@sabrinatoro just one question for you, but otherwise your PR looks good!

File: src/ontology/imports/merged_import.owl (hunk 1 above)
Comment (sabrinatoro) [reply src/ontology/imports/merged_import.owl:25068]: I think it is something that was removed from RO.

Comment (sabrinatoro) [reply src/ontology/imports/merged_import.owl:25068]: I think this is a note that was in one of the RO term that was removed

**Review by katiermullen — APPROVED** [review]

**Pull request merged**